lib/Conversation/CommentForm/CommentFormActions.js

```diff
@@ -1,31 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '../../Button';
-import MentionIcon from '../../../assets/icons/mention.svg';
+import Icon from '../../Icon';
 
-const CommentFormActions = props => (
-  <div className="comment-form__actions">
-    <Button
-      clickHandler={() => {}}
-      types={['clear', 'size-sm']}
-      className="form__button"
-    >
-      <MentionIcon />
-    </Button>
+const CommentFormActions = props => {
+  const isSubmittedClass = props.submitted ? 'is-submitted' : '';
 
-    <span className="comment-form__actions comment-form__actions--submit">
-      <Button types={['danger', 'size-sm']} clickHandler={props.onCancel}>
-        Cancel
-      </Button>
-      <Button types={['secondary', 'size-sm']} clickHandler={() => {}} isSubmit>
-        Send
-      </Button>
-    </span>
-  </div>
-);
+  return (
+    <div className={`comment-form__actions ${isSubmittedClass}`}>
+      <span className="comment-form__actions comment-form__actions--submit">
+        <Button types={['danger', 'size-sm']} clickHandler={props.onCancel}>
+          Cancel
+        </Button>
+        <Button
+          types={['secondary', 'size-sm']}
+          disabled={!props.hasValue}
+          clickHandler={() => {}}
+          isSubmit
+        >
+          {props.submitted && !props.existingConversation ? (
+            <Icon name="loader" />
```

Comment:
There is an existing component for handling showing progression when submitting. If you check out `ProgressButton` if might already provide you what you need 👍